aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/CallbackContext.java

```diff
@@ -21,9 +21,11 @@ public class CallbackContext extends StdCallbackContext implements TaggingContex
     private boolean storageAllocated;
     private boolean allocatingStorage;
     private boolean readReplicaPromoted;
+    private boolean automaticBackupReplicationStopped;
+    private boolean automaticBackupReplicationStarted;
+    private String dbInstanceArn;
```

Comment:
Is Arn serializable? If so, we might consider using it instead of a plain string, given the amount of string operations happening with it later on in the code.